docs/designers-developers/developers/tutorials/javascript/js-build-setup.md

```diff
@@ -119,13 +119,13 @@ To configure npm to run a script, you use the scripts section in `package.json`
 
 You can then run the build using: `npm run build`.
 
-After the build finishes, you will see the built file created at `build/index.js`. Enqueue this file in the admin screen as you would any JavaScript in WordPress, see [Step 2 in this tutorial](loading-javascript.md), and the block will load in the editor.
+After the build finishes, you will see the built file created at `build/index.js`. Enqueue this file in the admin screen as you would any JavaScript in WordPress, see [Step 2 in this tutorial](readme.md), and the block will load in the editor.
```

Comment:
I recommend we keep the link to the relevant content. So how about we change the link text instead of the link itself. Here is my suggestion:

```suggestion
After the build finishes, you will see the built file created at `build/index.js`. Enqueue this file in the admin screen as you would any JavaScript in WordPress, see [Loading JavaScript page for details](loading-javascript.md), and the block will load in the editor.
```

Reply:
Hi @mkaz I rebased and applied the suggested change.

Reply:
@mkaz, @jorgefilipecosta Is there anything you guys need from me?